jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/python/PythonLocalLamdaRunConfigurationIntegrationTest.kt

```diff
@@ -64,7 +64,7 @@ class PythonLocalLamdaRunConfigurationIntegrationTest(private val runtime: Runti
             import os
 
             def lambda_handler(event, context):
-                print(os.environ)
+                print os.environ
```

Comment:
Is this okay? Looks like we set some breakpoints and verify output for all the python runtimes we support.